cohere/responses/cluster.py

```diff
@@ -65,39 +67,40 @@ def __init__(
         self.output_outliers_url = output_outliers_url
         self.clusters = clusters
         self.meta = meta
+        self.error = error
 
     @classmethod
-    def from_dict(cls, data: Dict[str, Any], meta: List[Meta]) -> 'ClusterJobResult':
+    def from_dict(cls, data: Dict[str, Any]) -> 'ClusterJobResult':
         if data.get('clusters') is None:
             clusters = None
         else:
             clusters = [Cluster.from_dict(c) for c in data['clusters']]
-
         return ClusterJobResult(
             job_id=data['job_id'],
             status=data['status'],
             output_clusters_url=data['output_clusters_url'],
             output_outliers_url=data['output_outliers_url'],
             clusters=clusters,
-            meta=meta
+            error=data.get('error'),
+            meta=data['meta']
         )
 
 
 class CreateClusterJobResponse(CohereObject):
     job_id: str
-    meta: List[Meta]
+    meta: Meta
 
-    def __init__(self, job_id: str, meta: List[Meta], wait_fn):
+    def __init__(self, job_id: str, meta: Meta, wait_fn):
```

Comment:
this is a my bad, but this should be optional. all meta tags are optional since there are cases when it might not be sent